smsapi/Api/Action/Base.cs

```diff
@@ -8,37 +8,27 @@
 
 namespace SMSApi.Api.Action
 {
-    public abstract class Base<T, TResult>
+    public abstract class Base<T>
     {
-        protected IClient client;
-        protected Proxy proxy;
+        private Proxy proxy;
 
         protected virtual RequestMethod Method => RequestMethod.POST;

```

Comment:
I would rather avoid default method types and declare them visible in each action.

Reply:
cca3fdb